src/FlipCounter.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import "./FlipCounter.css";
+
+class Counter extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+     x:"000000"
+    };
+  }
+  add = () => {
+    
+    this.setState({ x:"00000"+ parseInt(parseInt(this.state.x) + 1) });
```

Comment:
You do not need double parseInt here

Additionally, this function will not work after 10. It will make 0000011